tests/test_candidate_generation.py

```diff
@@ -31,7 +31,7 @@ def test_candidate_generation(self):
         results = candidate_generator(['(131)I-Macroaggregated Albumin'], 10)
 
         canonical_ids = [x.concept_id for x in results[0]]
-        assert canonical_ids == ['C0000005', 'C0000102', 'C0000084']
```

Comment:
wait why did this bit change? You added 'C0000015' right? Why did other stuff change?

Reply:
Ummm, i'm going to say this is a quirk of the ANN thing?

Reply:
hmm, perhaps. thats a bit odd given the size of the kb. maybe it performs strangely on small kbs

Reply:
hmm, perhaps. thats a bit odd given the size of the kb. maybe it performs strangely on small kbs

Reply:
I think this is the case - the index in the tests is particularly unstable because the TF-IDF vectorizer contains very few char-3grams, meaning additions of any new aliases or entities could substantially change similarity scores between entities. (I checked this by adding some random aliases to the entity I added and the results changed again.) This isn't ideal for a test but I don't have a good alternative either....

Reply:
So yeah, I think it's not the ANN thing, but more that similarity between TF-IDF vectors induced from a tiny corpora doesn't make too much sense.

Reply:
seems fine 👍